slackbot/slackclient.py

```diff
@@ -61,10 +61,11 @@ def parse_slack_login_data(self, login_data):
         self.login_data = login_data
         self.domain = self.login_data['team']['domain']
         self.username = self.login_data['self']['name']
-        self.parse_user_data(login_data['users'])
-        self.parse_channel_data(login_data['channels'])
-        self.parse_channel_data(login_data['groups'])
-        self.parse_channel_data(login_data['ims'])
+        self.parse_user_data(self.webapi.users_list()['members'])
+        self.parse_channel_data(self.webapi.conversations_list(
+            exclude_archived=True,
+            types="public_channel,private_channel"
+        )['channels'])
```

Comment:
```suggestion
        for page in self.webapi.users_list(limit=200):
            self.parse_user_data(page['members'])
        for page in self.webapi.conversations_list(exclude_archived=True,
                                                   types="public_channel,private_channel",
                                                   limit=200):
            self.parse_channel_data(page['channels'])
```

to iterate through all pages

Reply:
@amuraru you are correct - we should implement pagination, though I believe we need to use the `cursor` parameter, passing the `next_cursor` value obtained from the `response_metadata` - I will work on this

Reply:
@amuraru I implemented pagination, but slightly differently, to account for rate limit errors